VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -572,4 +572,8 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
     leaveHearing() {
         this.hostWantsToJoinHearing = false;
     }
+
+    shouldMuteHearing(): boolean {
+        return super.shouldMuteHearing() && !this.hostWantsToJoinHearing;
```

Comment:
Using hostWantsToJoinHearing as an existing indicator.

Me and @ChrisPaulP  went over this earlier and it works in the scenarios we would expect but there may be some high cognitive complexity introduced. We can discuss if needed.

Reply:
Actually I think this would result in the video being unmuted while joining but relies on other logic that prevents the video being shown. I will look at something that makes it more reliable as a unit